page/studentlist.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from 'preact/hooks'
+import { Main } from '../component/elements.jsx'
+import { Table } from '../component/table.jsx'
+import { users } from '../data/users.js'
+
+export const StudentList = () => {
+  const [data, setData] = useState([])
+
+  const columns = [
+    'sid',
+    'login',
+    'name',
+    'discordId',
+    'email',
+    'avatar',
+    'speciality',
+  ]
```

Comment:
I think this way of describing columns is a bit to restrictive, we want more:
each column could specify:
- prefered with
- a color or component to render it's content
- a column display different that it's key (we could use an icon or something, have accent, and capitalized version...)
- if it's the primary key (used for key'ing the rows !)
- type of the expected value maybe could be helpfull to hint how to display it